schedule/functional/extra_tests_textmode.yaml

```diff
@@ -100,6 +100,7 @@ conditional_schedule:
 schedule:
     - installation/bootloader_start
     - boot/boot_to_desktop
+    - console/ansible
```

Comment:
schedule right after `tar` tests, do note that Tumbleweed also schedules salt, and I'm not sure if that would bring unwanted consequences

Reply:
I'm sure it will not and if yes then I will fix this test module to behave nicely :-)